frontend-next.js

```diff
@@ -0,0 +1,5 @@
+const nextConfig = require('eslint-config-next');
+
+module.exports = {
+  extends: [...nextConfig],
```

Comment:
I would put the rest of frontend typescript config here as well, so I don't need to bother with all the configuration on nextjs project, but only have 1